docs/user-guide/examples/dark-vessel-detection.mdx

```diff
@@ -463,45 +485,50 @@ We can now use the [`odc` package](https://odc-stac.readthedocs.io/en/latest/int
 
 We'll get an [`xarray.Dataset` object](https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html) back that we can simply open & return as a `uint8` array:
 
-```python {5,10,27-33,35-36,38} showLineNumbers
+```python {5,13,33-39,41-43} showLineNumbers
 @fused.udf
 def udf(
-    bbox: fused.types.ViewportGDF,
+   bbox: fused.types.Bounds,
 ):
-    import odc.stac
-    import planetary_computer
-    import pystac_client
-
-    time_of_interest="2024-09-03/2024-09-04"
-    bands = ["vv"]
-    bbox = gpd.GeoDataFrame({}, geometry=[shapely.box(-93.90425364,  29.61879782, -93.72767384,  29.7114987 )], crs=4326)
-
-    catalog = pystac_client.Client.open(
-        "https://planetarycomputer.microsoft.com/api/stac/v1",
-        # Details as to why we need to sign it are addressed here: https://planetarycomputer.microsoft.com/docs/quickstarts/reading-stac/#Searching
-        modifier=planetary_computer.sign_inplace,
-    )
-
-    items = catalog.search(
-        collections=["sentinel-1-grd"],
-        bbox=bbox.total_bounds,
-        datetime=time_of_interest,
-        query=None,
-    ).item_collection()
-    print(f"{len(items)=}")
-
-    ds = odc.stac.load(
-        items,
-        crs="EPSG:3857",
-        bands=bands,
-        resolution=10, # We want to use the native Sentinel 1 resolution which is 10m
-        bbox=bbox.total_bounds,
-    ).astype(float)
-
-    da =  ds[bands[0]].isel(time=0)
-    image = da.values * 1.0
-
-    return image.astype('uint8')
+   import odc.stac
```

Comment:
Very much a nitpick, but it seems you changed the indentation from 4 to 3 spaces. I assume the editor might do that because you are in a mdx file, but for python code snippet that is a bit strange to do

Reply:
(this also causes the big diffs I think)

Reply:
Yes, I guess the editing of python code within mdx format caused this, I will try to configure my VSCode for this.